packages/@aws-cdk/aws-iot-actions-alpha/lib/step-functions-start-state-machine-action.ts

```diff
@@ -0,0 +1,65 @@
+import * as iam from 'aws-cdk-lib/aws-iam';
+import * as iot from '@aws-cdk/aws-iot-alpha';
+import * as stepfunctions from 'aws-cdk-lib/aws-stepfunctions';
+import { CommonActionProps } from './common-action-props';
+import { singletonActionRole } from './private/role';
+import { ArnFormat, Stack } from 'aws-cdk-lib';
+
+/**
+ * Configuration properties of an action for the Step Functions State Machine.
+ */
+export interface StepFunctionsStartStateMachineActionProps extends CommonActionProps {
+  /**
+   * Name of the state machine execution prefix.
+   * The name given to the state machine execution consists of this prefix followed by a UUID. Step Functions creates a unique name for each state machine execution if one is not provided.
+   *
+   * @see https://docs.aws.amazon.com/iot/latest/developerguide/stepfunctions-rule-action.html#stepfunctions-rule-action-parameters
+   *
+   * @default: None
```

Comment:
I see you have mentioned,
```
Step Functions creates a unique name for each state machine execution if one is not provided.
```

This can be added to the `default -`.